public/app/actions/actions.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+var Reflux = require('reflux');
+
+var actions = Reflux.createActions([
```

Comment:
We will have tons of actions. It might be good to have multiple, separated action files. See https://github.com/yetu/controlcenter/pull/2/files

However this can be refactored later.